src/main/java/com/orasi/bluesource/AddRoleType.java

```diff
@@ -0,0 +1,179 @@
+	@FindBy(xpath = "//*[@id=\'new_role\']/div[1]/div/div") private Element billingLbl;
+	@FindBy(css = "table.table-striped.table-hover") private Webtable projectsTable;
+	@FindBy(xpath = "//*[@id=\"accordion\"]/div/div[13]/div/div[1]/button[1]") private Button newRoleButton;
+	@FindBy(xpath = "//*[@id=\"new_role\"]/div[1]/div/div") private Label lblBilling;
+	@FindBy(id = "role_role_type_id") private Listbox listRoleTypes;
+	@FindBy(xpath = "//input[@value='Create Role']") private Button btnAssignRoleType;
+	
+	private String newRoleTypeName;
+
+	public AddRoleType(OrasiDriver driver){
+		this.driver = driver;
+		ElementFactory.initElements(driver, this);
+	}
+
+	public void openRoleType() {
+
+		if (lnkAdminTab.isDisplayed() == true)
+		{
+			lnkAdminTab.click();
+			lnkRoleTypes.click();
+		}
+	}
+
+	public void click_AddNewRoleTypeButton() {
+
+		if (lnkAddNewRoleTypeButton.isDisplayed() == true)
+		{
+
+			lnkAddNewRoleTypeButton.click();
+		}
+	}
+
+	public void createNewRole(String roleType) {
+		
+		this.newRoleTypeName = roleType;
+
+		if (txtRoleName.isDisplayed()==true) {
+			txtRoleName.sendKeys(newRoleTypeName);
+			billableCheckBox.uncheck();
+
+			Select select = new Select(driver.findElement(By.name("role_type[permission]")));
+			select.selectByValue("View");
+
+			btnCreateRoleType.click();
+		}
+	}
+
+
+	public boolean verifyNewRole() {
+
+		Boolean verify = true;
+
+		if (rolesTable.isDisplayed() == verify) {
+
+			if (rolesTable.getRowWithCellText(newRoleTypeName) > 0) {		
+				System.out.println("New Role Type is created");
+			}
+			else {
+				System.out.println("New role type is not created");
+				verify = false;
+			}
+		}
+
+		return verify;
+	}
+
+	public int getProjectsTableRows() {
+		int rowCount = 0;
+		try
+		{
+			rowCount = projectsTable.getRowCount();
+		}
+		catch(NullPointerException e)
+		{
+			System.out.println("Null pointer exception, no accounts found  \n" + e.getLocalizedMessage());
+		}
+		return (rowCount);
+	}
+
+	public void openFirstProject() {
+		if (projectsTable.isDisplayed() == true) {
+
+			if (getProjectsTableRows() > 0) {
+				projectsTable.getCell(2, 1).findElement(
+						By.xpath("//*[@id=\'panel_body_1\']/div/table/tbody/tr[1]/td[1]/a")).click();
+			}
+			else {
+				System.out.println("Projects not found");
+			}
+		}
+	}
+
+	public void clickNewRole() {
+		if (newRoleButton.isDisplayed() == true) {
+			newRoleButton.click();
+		}
+		else {
+			System.out.println("New Role Button was not found");
+		}
+	}
+
+	public void toggleBilling() {
+		if (lblBilling.isEnabled() == true) {
+			String str = lblBilling.getAttribute("class");
+			if (str.contains("toggle btn btn-xs btn-primary")) {
+				lblBilling.jsClick();
+				System.out.println("New Label is : " + lblBilling.getAttribute("class"));
+			}
+			else {
+				System.out.println("No need to change because it is already: " 
+							+ lblBilling.getAttribute("class"));
+			}
```

Comment:
similar comments about console logging.  If you are just debugging, then please remove once your done.  If you want that info displayed in report, then use TestReporter.log("Your message here");